classes/UpgradeTools/TaskRepository.php

```diff
@@ -58,8 +57,6 @@ public static function get(string $step, UpgradeContainer $container): AbstractT
     {
         switch ($step) {
             // MISCELLANEOUS (upgrade configuration, checks etc.)
-            case TaskName::TASK_CHECK_FILES_VERSION:
-                return new CheckFilesVersion($container);
```

Comment:
This task is used in the current UI. If you plan to remove it, I should find a call to this task being removed as well.

Reply:
This call is made on the upgrade.js javascript script. Do we want to waste our time updating this file when it is going to be deleted?

Reply:
I think so. Instead of having an error leaving us confused about feature supposed to be working or not, we remove it from the old UI.